### DISCUSSION TITLE
How to add scheme to a URL without scheme

### DISCUSSION THREAD

**Discussion by z0z0r4** [aio-libs/yarl#815] [Q&A] [answered]
I have a URL, which is `URL("google.com")`
I want to add a scheme to it. I need `URL("https://google.com")`
But how...
I try `URL("google.com").with_scheme("https")`, raise ValueError("scheme replacement is not allowed for relative URLs")
Any method to do it?
![image](https://user-images.githubusercontent.com/78744121/214837305-36858c0f-843d-44f6-a3ff-92d2fe5b13cd.png)

**Comment by Vizonex** [comment, marked as answer]
Sorry if this is late but have you tried `URL.build`?
```python
from yarl import URL
print(URL.build(host="google.com", scheme="https"))
```

**Reply by z0z0r4** [reply]
Thank for your answer~!

~~It works, but I have forgotten what I was doing at that time, but no problem, hiahia~~
